applications: nrf5340_audio: Revert to adv after short burst of dir_adv (#13898)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

nRF5340 Audio
-------------

> * Added support for Filter Accept List.

Reviewer: Suggest we add some more info. E.g. that this is enabled by default and how it influences the user experience.

* Updated:

>   * ISO data sending has been refactored, and is now done in a single file: bt_le_audio_tx.

Reviewer 2: This and other edits in this section are addressing the changelog formatting, where "Updated" needs to match the sentences in the bullets.
```suggestion
  * Sending of the ISO data, which is now done in a single file :file:`bt_le_audio_tx`.
```

Author: This is strictly not part of this PR, same for the comment below

Reviewer 2: OK, I'll open a separate PR...

Author: Don't need to, I've changed them here now.

>   * Split the generic applications into their own samples with separate main.c files.

Reviewer 2: OK, I'll open a separate PR...

>   * Split the generic applications into their own samples with separate main.c files.

Author: Don't need to, I've changed them here now.

>   * Revert back to slower advertising after a short burst (1.28s) of directed advertising.

Author: That sentence doesn't make sense. Is the thought that the full sentence is: "Updated advertising process by revert...."?

>   * Revert back to slower advertising after a short burst (1.28s) of directed advertising.

Reviewer 2: Yes, as I wrote above "This and other edits in this section are addressing the changelog formatting, where "Updated" needs to match the sentences in the bullets".

>   * Revert back to slower advertising after a short burst (1.28s) of directed advertising.

Author: Ok, will change. Is that a new thing? I looked through older release notes, and didn't see that any of them were written like that.
